plugins/out_s3/s3.c

```diff
@@ -1081,7 +1074,7 @@ static int upload_data(struct flb_s3 *ctx, struct s3_file *chunk,
 multipart:
 
     if (init_upload == FLB_TRUE) {
-        m_upload = create_upload(ctx, tag, tag_len);
+        m_upload = create_upload(ctx, tag, tag_len, chunk->first_log_time);
```

Comment:
Same question here, are we guaranteed to have a chunk?

Reply:
Before I think we were passing the first_log_time from cb_s3_flush to here- why was that removed?